winlogbeat/eventlog/wineventlog.go

```diff
@@ -197,9 +197,9 @@ func (l *winEventLog) openChannel(bookmark win.EvtHandle) error {
 	// https://msdn.microsoft.com/en-us/library/windows/desktop/aa385771(v=vs.85).aspx#pull
 	signalEvent, err := windows.CreateEvent(nil, 0, 0, nil)
 	if err != nil {
-		return nil
+		return err
```

Comment:
@andrewkroh Please confirm that this is a correct change; the sys packages have no docs, so it could be that you intended this to be a silent no-op on failure, but I have guessed the contrary. Neither option fails tests.

Reply:
👍